### DISCUSSION TITLE
autodiscoverfilter question

### DISCUSSION THREAD

**Discussion by Rbsn-joses** [renovatebot/renovate#9353] [Request Help] [answered]
good afternoon, i'm trying to use the autodiscoverfilter by args on kubernetes, i would like to know how to use it to update repositories of a certain group by cli.
I was not able to understand this part of the documentation very well

**Comment by Rbsn-joses** [comment, marked as answer]
I managed to solve it, I didn't pay attention to the variable RENOVATE_AUTODISCOVER has to be true

**Reply by viceice** [reply]
Yes, autodiscover is disabled by default.
